packages/devtools_app/lib/src/screens/performance/performance_controller.dart

```diff
@@ -131,6 +137,28 @@ class PerformanceController extends DisposableController
           }
         }),
       );
+    } else {
+      final shouldLoadOfflineData = offlineController
```

Comment:
this was copied from performance_screen.dart, where it used to live in didChangeDependencies